src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -48,29 +88,291 @@ struct miopen_convolution
     template <class Self, class F>
     static auto reflect(Self& self, F f)
     {
-        return pack(f(self.op.padding, "padding"),
```

Comment:
I think we can retain these op fields since they're shared between conv, deconv, and quant conv.

Reply:
`pack_join(pack(f(self.op, "op")),` would print all attributes for any of the conv types.

Reply:
I am running into errors when trying to reflect entire `"op"` with templated code.  Errors says it doesn't have matching `stream_write_value_impl(rank<XY>, std::ostream& os, const T& x) ` for either `op::deconvolution`, `op::convolution` or `op::quant_convolution`.

Reply:
Made  required changes.